docs/AdviceForOssBeginners.md

```diff
@@ -0,0 +1,38 @@
+# Advice for OSS Beginners
+
+## Pay attention to instructions
+
+Take your time to learn and follow the workflow to the letter. 
+
+* If the setup instructions mention a specific version of a software to use, **use that exact version** 
+  instead of the latest version.
+* Do not skip any steps because you think that step is "not important". 
+* Do not assume current project does things the same way things were one in that _other_ project you are familiar with.
```

Comment:
"that _other_ project" seems like we're trying to imply something. "other projects" is more tone-neutral.